iphone/Maps/Maps.xcodeproj/xcshareddata/xcschemes/OMaps.xcscheme

```diff
@@ -26,8 +26,34 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES">
+      shouldUseLaunchSchemeArgsEnv = "NO">
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "ED097E752BB80C320006ED01"
+            BuildableName = "OMapsTests.xctest"
+            BlueprintName = "OMapsTests"
+            ReferencedContainer = "container:Maps.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-IsTests"
```

Comment:
On the second thought, we already know if tests are launched either from XCode or from command line, even without that parameter, right?

Reply:
Yes! This launch arg is used only during the test run started by both xcode and console.

Reply:
Maybe make it more explicit, like -disableDrape ?